core/src/main/java/io/questdb/cairo/GeoHashes.java

```diff
@@ -179,15 +188,38 @@ public static long fromString(CharSequence hash, int start, int end) throws Nume
     }
 
     private static long appendChar(long geohash, char c) throws NumericException {
+        byte idx;
         if (c >= 48 && c < 123) { // 123 = base32Indexes.length + 48
-            byte idx = base32Indexes[c - 48];
+            idx = base32Indexes[c - 48];
             if (idx >= 0) {
                 return (geohash << 5) | idx;
             }
         }
         throw NumericException.INSTANCE;
     }
 
+    public static boolean isValidChars(CharSequence tok, int start) {
+        int idx;
+        for (int i = start, n = tok.length(); i < n; i++) {
+            idx = tok.charAt(i);
+            if (idx < 48 || idx >= 123 || base32Indexes[idx - 48] == -1) {
```

Comment:
minor, we believe that > works faster than >= so it's better to write `> 122` instead of `>= 123`